src/main/kotlin/no/nav/syfo/sykmelding/model/SykmeldingDTO.kt

```diff
@@ -24,5 +24,6 @@ data class SykmeldingDTO(
     val behandletTidspunkt: OffsetDateTime,
     val behandler: BehandlerDTO,
     val syketilfelleStartDato: LocalDate?,
-    val navnFastlege: String?
+    val navnFastlege: String?,
+    val egenmeldt: Boolean
```

Comment:
Den burde ikke være påkrevd her, bare å legge til ? på slutten

Reply:
Jeg er ikke helt stødig i Kotlin, men blir den ikke default false hvis den ikke blir eksplisitt satt i SykmeldingDBModel.toSykmeldingDTO()? Som i Java? Er det i så tilfelle noen fare ved å alltid inkludere den?

Reply:
Men det spiller forsåvidt ingen rolle, jeg fikser :) Takk for input!